gobblin-modules/gobblin-http/src/main/java/gobblin/writer/AvroHttpWriterBuilder.java

```diff
@@ -36,4 +38,8 @@ public AvroHttpWriterBuilder fromConfig(Config config) {
     this.responseHandler = new HttpResponseHandler();
     return this;
   }
+
+  public String getLimiterKey (State state) throws NotConfiguredException {
+    return AvroHttpWriterBuilder.class.getName();
```

Comment:
@ibuenros @zxcware , I'm using url.getProtocol() + url.getPath() as the key name.